webapp/channels/src/components/pricing_modal/card.tsx

```diff
@@ -58,19 +68,77 @@ const StyledDiv = styled.div<StyledProps>`
 background-color: ${(props) => props.bgColor};
 `;
 
+export function BlankCard() {
+    const {formatMessage} = useIntl();
+    const [, contactSalesLink] = useOpenSalesLink();
+
+    return (
+        <div className='BlankCard'>
+            <div>
+                <BlankCardImage/>
+            </div>
+
+            <div className='description'>
+                <div className='title'>
+                    <span className='questions'>
+                        {formatMessage({id: 'pricing_modal.questions', defaultMessage: 'Questions?'})}
+                    </span>
+                    <span className='contact'>
+                        <ExternalLink
+                            location='cloud_pricing_modal'
+                            href={contactSalesLink}
+                        >
+                            {formatMessage({id: 'pricing_modal.contact_us', defaultMessage: 'Contact us'})}
+                        </ExternalLink>
+                    </span>
+                </div>
+                <div className='content'>
+                    {formatMessage({id: 'pricing_modal.reach_out', defaultMessage: 'Reach out to us and we’ll help you decide which plan is right for you and your organization.'})}
+                </div>
+            </div>
+            <div className='self-hosted-interest'>
+                <span className='interested'>
+                    {formatMessage({id: 'pricing_modal.interested_self_hosting', defaultMessage: 'Interested in self-hosting?'})}
+                </span>
+                <span className='learn'>
+                    <ExternalLink
+                        location='cloud_pricing_modal'
+                        href={HostedCustomerLinks.SELF_HOSTED_BILLING}
+                    >
+                        {formatMessage({id: 'pricing_modal.learn_more', defaultMessage: 'Learn more'})}
+                    </ExternalLink>
+                </span>
+            </div>
+        </div>
+    );
+}
+
 function Card(props: CardProps) {
     const {formatMessage} = useIntl();
+
+    const cloudFreeDeprecated = useSelector(deprecateCloudFree);
+    const isCloud = useSelector(isCurrentLicenseCloud);
```

Comment:
Wondering if we can move this out of this generic `Card` component and pass this through the props to follow the same model we currently have